src/stage0.txt

```diff
@@ -12,8 +12,8 @@
 # stable release's version number. `date` is the date where the release we're
 # bootstrapping off was released.
 
-date: 2021-02-14
-rustc: beta
+date: 2021-03-17
+rustc: nightly
```

Comment:
We'll have to wait for the bootstrap compiler to be upgraded before we can actually merge this.